Library/Formula/meanwhile.rb

```diff
@@ -0,0 +1,299 @@
+class Meanwhile < Formula
+  desc "Implementation of Lotus Sametime Community Client protocol"
+  homepage "http://meanwhile.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/meanwhile/meanwhile/1.0.2/meanwhile-1.0.2.tar.gz"
+  sha256 "38a8059eaa549cbcb45ca0a5b453e9608ceadab360db2ae14581fb22ddabdf23"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  patch :DATA
```

Comment:
The Linux distros tend to be happier to patch abandoned software. I'll let some of the other maintainers express their thoughts on including this or otherwise.